src/node_modules/_file-explorer.js

```diff
@@ -0,0 +1,292 @@
+var setval = require('setval')
```

Comment:
ok.
Just to maybe again give the reason for why a manually maintained nested `node_modules` directory is useful.
* It allows to do non-path-relative require calls so a search&replace is easier in case of refactoring
* It is easier to `npm publish` internaly modules to npm or elsewhere in case they grow into stuff that is useful across projects
* It enables to add a custom `package.json` which can contain a `"browserify"` field that includes all the transforms browserify will use (e.g. babel-transforms)

In general, running browserify transforms globally with the `-g` flag can break stuff, because it might try to run transforms on files that are not prepared. Each module should know the transform's it needs and list them in the package.json. Browserify will respect that and only execute only the relevant transforms on each module when it's traversing the tree to bundle everything.

I can anyway put it in `src/app` because for now there are no custom transforms for this module.

But the reason for currently using `babelify` and then on top of this also using `babel` is, because of trouble with global transforms.